l5kit/l5kit/environment/utils.py

```diff
@@ -74,35 +74,6 @@ def calculate_kinematic_rescale_params(sim_dataset: SimulationDataset) -> Kinema
     :param sim_dataset: the input dataset to calculate the action rescale parameters
     :return: the unnormalized action
     """
-    v_component_frames = []
```

Comment:
This will be only a one line function now, would it make sense to remove it and instantiate the class directly when it is needed ?

Reply:
Removing the function.